Src/MoneyFox.Windows/Views/UserControls/CategoryListUserControl.xaml.cs

```diff
@@ -54,7 +54,7 @@ private void DeleteCategory(object sender, RoutedEventArgs e)
                 return;
             }
 
-            ((AbstractCategoryListViewModel) DataContext).DeleteCategoryCommand.Execute(category);
```

Comment:
This seems to cause the following exception when you click on edit or delete in the select category dialog.

`Unable to cast object of type 'MoneyFox.Shared.ViewModels.SelectCategoryListViewModel' to type 'MoneyFox.Shared.ViewModels.CategoryListViewModel'.`